README.md

```diff
@@ -36,8 +36,9 @@
 
 The friendly Operating System for IoT!
 
-RIOT is a real-time multi-threading operating system that supports a range of
-devices that are typically found in the Internet of Things (IoT):
+RIOT is an open-source microcontroller operating system, designed to match
+the requirements of Internet of Things (IoT) devices and other embedded devices. 
```

Comment:
```suggestion
the requirements of Internet of Things (IoT) devices and other embedded devices.
```

Reply:
`git add -u` vs `git commit -a`  `git commit -u` doesn't do the right thing

Reply:
it was a relay resilient whit space